metadata-protobuf/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@joystream/metadata-protobuf",
-  "version": "2.5.0",
+  "version": "2.6.0",
```

Comment:
Lets bump this to `v3.0.0` to allow for version updates on carthage/mainnet in the `2.x.x` range.
And of course update the dependent packages.

Reply:
Sure. btw moving forward, wouldn't it be a better solution to have something like `2.6.0-beta` for pre-production /testing releases instead of reserving some range for a specific release, i.e., `2.x.x` range for carthage/mainnet. This way, devs can be more open to following semantic versioning guidelines properly, i.e., bumping major versions too in case of breaking changes/new features

Reply:
ok makes sense.

Reply:
Sorry, but I haven't made the version bump change yet; I was just confirming if it's fine to do it this way before making the change. So will you reopen this PR, Or should I create a separate PR including the version change, i.e., setting `@joystream/metadata-protobuf` version to `2.6.0-beta`